proof.go

```diff
@@ -11,8 +11,6 @@ import (
 	"github.com/iden3/go-merkletree-sql/v2"
 )
 
-var ErrNodeNotFound = errors.New("node not found")
```

Comment:
We should not remove such variables because they are exported and we don't change the major version, but we should deprecate it